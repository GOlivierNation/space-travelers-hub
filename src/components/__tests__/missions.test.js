import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../../redux/configureStore';
import Missions from '../missions';

describe('Tests for the Missions page', () => {
  it('Should render the Missions page', () => {
    const tree = renderer.create(<Provider store={store}><Missions /></Provider>).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('The Missions page should render mission', () => {
    render(<Provider store={store}><Missions /></Provider>);
    const mission = screen.getByText('Status');
    expect(mission).toBeInTheDocument();
  });
});
