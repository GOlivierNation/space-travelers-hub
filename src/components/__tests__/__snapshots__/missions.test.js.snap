// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for the Missions page Should render the Missions page 1`] = `
<table>
  <thead>
    <tr>
      <th
        className="mission-name"
      >
        <h3>
          Mission
        </h3>
      </th>
      <th
        className="mission-description"
      >
        <h3>
          Description
        </h3>
      </th>
      <th
        className="mission-status"
      >
        <h3>
          Status
        </h3>
      </th>
      <th>
         
      </th>
    </tr>
  </thead>
  <tbody />
</table>
`;
